transport/network/src/ping.rs

```diff
@@ -199,7 +199,7 @@ where
     /// # Arguments
     ///
     /// * `peers` - A vector of PeerId objects referencing the peers to be pinged
-    #[tracing::instrument(level = "info", skip(self))]
+    #[tracing::instrument(level = "info", skip(self, peers))]
     async fn ping(&self, peers: Vec<PeerId>) {
```

Comment:
Ping should return `Result<()>` and when empty peer list is input, it should be treat as an error:
- in the automatic heartbeat round the error is simply swallowed, as this is an allowed scenario
- in manual ping, this error should be propagated upwards

This is taken care of in this PR: https://github.com/hoprnet/hoprnet/pull/6486